assets/css/style.css

```diff
@@ -119,6 +122,210 @@ header {
   height: 150%;
 }
 
+.product-categories {
+  display: flex;
+  column-gap: 8px;
+  padding: 0;
+}
+
+.type_dev {
+  display: flex;
+  column-gap: 8px;
+  padding: 0;
+}
+
+.article_text {
+  width: 95%;
+  margin: 0 auto;
+}
+
+.article_img {
+  width: 95%;
+  margin: 0 auto;
+}
+
+.button-category {
+  padding: 10px 15px;
+  border: none;
+  border-radius: 10px;
+  color: #6070ff;
+  background-color: #ebebff;
+}
+
+.li-color {
+  color: #344563;
+}
+
+.div-works {
+  /* display: flex;
+  flex-direction: column; */
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+
+  /* border: 3px solid; */
```

Comment:
- please i will kindly suggest you remove all commented code from your code base as it is unprofessional to leave commented code.This is not in line with clean code practice :+1: